test_mq_service.py

```diff
@@ -200,21 +205,22 @@ def _base_add_host_test(self, host_data, expected_results, host_keys_to_check):
 
 
 class MQhandleMessageTestCase(MQAddHostBaseClass):
-    def test_handle_message_verify_metadata_pass_through(self):
-        request_id = str(uuid.uuid4())
-        expected_insights_id = str(uuid.uuid4())
-
-        metadata = {"request_id": request_id, "archive_url": "https://some.url"}
-
-        host_data = {
+    def _host_data(self):
+        return {
             "display_name": "test_host",
-            "insights_id": expected_insights_id,
+            "id": str(uuid.uuid4()),
+            "insights_id": str(uuid.uuid4()),
             "account": "0000001",
             "stale_timestamp": "2019-12-16T10:10:06.754201+00:00",
             "reporter": "test",
         }
 
-        message = {"operation": "add_host", "platform_metadata": metadata, "data": host_data}
+    def test_handle_message_verify_metadata_pass_through(self):
+        request_id = str(uuid.uuid4())
+
+        metadata = {"request_id": request_id, "archive_url": "https://some.url"}
```

Comment:
Just making this more concise.

```suggestion
        metadata = {"request_id": str(uuid.uuid4()), "archive_url": "https://some.url"}
```